resources/leiningen/new/kraken_http_api/service.clj

```diff
@@ -0,0 +1,35 @@
+(ns {{name}}.service
+  (:require [io.pedestal.http :as bootstrap]
+            [io.pedestal.http.route :as route]
+            [io.pedestal.http.route.definition :refer [defroutes]]
+            [io.pedestal.interceptor :refer [interceptor]]
+            [ring.util.response :as ring-resp]
+            [turbovote.resource-config :refer [config]]
+            [pedestal-toolbox.params :refer :all]
+            [pedestal-toolbox.content-negotiation :refer :all]
+            [kehaar.core :as k]
+            [clojure.core.async :refer [chan go alt! timeout]]))
+
+(def ping
```

Comment:
It's not something we need to solve now by any means. And maybe ping is fine to just stay there as a "http server is up" endpoint, but it would be nice to also have a way of knowing that this http server can actually serve it's purpose by talking with backends.